com.adobe.Flash-Player-Projector.yaml

```diff
@@ -0,0 +1,60 @@
+app-id: com.adobe.Flash-Player-Projector
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: flashplayer
+tags: [proprietary]
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host:ro
+  - --filesystem=xdg-run/gvfs:ro
+modules:
+  - shared-modules/gtk2/gtk2.json
+  - name: flashplayer
+    buildsystem: simple
+    build-commands:
+      - make install
+      - install -D -t /app/bin/ apply_extra
+      - install flashplayer /app/bin
+      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps/ com.adobe.Flash-Player-Projector.svg
+      - install -Dm644 -t /app/share/applications/ com.adobe.Flash-Player-Projector.desktop
+      - install -Dm644 -t /app/share/metainfo/ com.adobe.Flash-Player-Projector.appdata.xml
+      - |
+        for size in {32,64,128,256}; do
+          rsvg-convert com.adobe.Flash-Player-Projector.svg -w "${size}" -a -o "${size}.png";
```

Comment:
This could be a one-liner, running `mkdir -p /app/share/icons/hicolor/${size}x${size}/apps/ &&` before the rsvg-convert call that outputs the file directly in the installed location. Or it could be a rule in the Makefile :)